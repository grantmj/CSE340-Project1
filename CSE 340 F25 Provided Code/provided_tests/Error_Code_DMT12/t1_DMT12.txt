TASKS
    1 2 3
POLY
    F = x + 2;
    G = x - 1;
    F(Y,Z) = 3;
    G(Y,Z) = Y + (Y - 2) + Z;
EXECUTE
    INPUT A;
    w = F(A);
    OUTPUT w;
INPUTS
    3 4 5
