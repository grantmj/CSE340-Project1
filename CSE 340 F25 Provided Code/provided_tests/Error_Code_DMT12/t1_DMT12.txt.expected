Semantic Error Code DMT-12: 6 7 
